### DISCUSSION TITLE
what mip mean in the code

### DISCUSSION THREAD

**Discussion by neilgogogo** [NVlabs/instant-ngp#163] [Q&A] [answered]
Thank you for the excellent work, I have a question about the code.

I saw many **mip** in the code, what does mip mean?

https://github.com/NVlabs/instant-ngp/blob/1f400487e99938c97a0f090387754db03916b238/src/testbed_nerf.cu#L1022

Many thanks

**Comment by Tom94** [comment, marked as answer]
Hi there, `mip` is short for "mipmap level" and refers to the level of the multi-resolution occupancy grid that is to be used for empty-space skipping. (described in Appendix C of the paper)

**Reply by neilgogogo** [reply]
thanks, it makes sense:)
